loadgen/CMakeLists.txt

```diff
@@ -32,8 +32,7 @@ set(SOURCE
   ${CMAKE_CURRENT_SOURCE_DIR}/version.h
 )
 
-#execute_process(COMMAND python setup.py build
-#                WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
+execute_process(COMMAND python ${CMAKE_CURRENT_SOURCE_DIR}/version_generator.py ${CMAKE_BINARY_DIR}/version_generated.cc ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
How is "${CMAKE_BINARY_DIR}/version_generated.cc" compiled in to the binary? Does it need to be included in the "Add source files." section above?

Reply:
It's not at the moment. But that would make sense. I'm away until next week, but I can make this change then if not sooner.